File: patterns/javascript-patterns/pattern150.js
// Pattern 150: ...

// *   *
// * *
// *
// * *
// *   *

let n = 5;
let x = Math.floor(n / 2) + 1;

for (let i = 1; i <= n; i++) {
    let row = "";
    for (let j = 1; j <= Math.floor(n / 2) + 1; j++) {
        if (j === 1 || j === x) {
            row += "* ";
        } else {
            row += "  ";
        }
    }
    if (i <= Math.floor(n / 2)) {
        x--;
    } else {
        x++;
    }
    console.log(row);
}

File: patterns/python-patterns/pattern150.py
# Pattern 150: ...

# *   *
# * *
# *
# * *
# *   *

n = 5

x = n // 2 + 1

for i in range(1, n + 1):
    for j in range(1, n // 2 + 2):
        if j == 1 or j == x:
            print("*", end=" ")
        else:
            print(" ", end=" ")
    if i <= n // 2:
        x -= 1
    else:
        x += 1
    print()
